yt/frontends/halo_catalog/tests/test_outputs.py

```diff
@@ -44,7 +51,7 @@ def test_halo_catalog(self):
         fn = fake_halo_catalog(data)
         ds = yt_load(fn)
 
-        assert isinstance(ds, HaloCatalogDataset)
+        assert isinstance(ds, YTHaloCatalogDataset)
```

Comment:
Oh I missed this but in tests, checking for exact typing might be preferable (same applies a few lines bellow this)
```suggestion
        assert type(ds) is YTHaloCatalogDataset
```

Reply:
Good catch, I'll fix that.